Source/SwiftLintCore/Models/RuleConfigurationDescription.swift

```diff
@@ -390,12 +390,16 @@ public protocol InlinableOptionType: AcceptableByConfigurationElement {}
 ///            error: 2
 ///    ```
 @propertyWrapper
-public class ConfigurationElement<T: AcceptableByConfigurationElement & Equatable> {
+public struct ConfigurationElement<T: AcceptableByConfigurationElement & Equatable> {
```

Comment:
Should `AcceptableByConfigurationElement` itself have the `Equatable` requirement since it doesn't really seem useful to conform non-equatable values to that protocol?